mon-pix/app/components/training/card.js

```diff
@@ -33,17 +33,35 @@ export default class Card extends Component {
 
   get tagColor() {
     if (this.args.training.isAutoformation) {
-      return 'blue-light';
+      return 'primary';
     }
-    return 'purple-light';
+    if (this.args.training.isElearning) {
+      return 'success';
+    }
+    if (this.args.training.isHybrid) {
+      return 'error';
+    }
+    if (this.args.training.isInPerson) {
+      return 'secondary';
+    }
+    return 'tertiary';
   }
 
   get imageSrc() {
     const randomNumber = this._getRandomImageNumber();
     if (this.args.training.isAutoformation) {
-      return `/images/illustrations/trainings/Illu_Parcours_autoformation-${randomNumber}.png`;
+      return `/images/illustrations/trainings/Formation-Parcours_autoformation-${randomNumber}.svg`;
```

Comment:
J'ai l'impression que les noms de fichiers ont "Formation" en français et le type de formation en anglais sauf celui-ci qui a tout en français. 
Pour harmoniser correctement je propose de choisir une langue ou l'autre.

Reply:
J'ai laissé les noms fourni par Quentin mais why not

Reply:
J'ai changé tous les noms de fichier ♻️

Reply:
Cool